lms/envs/common.py

```diff
@@ -4146,8 +4146,11 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 ########################## BLOCKSTORE #####################################
 BLOCKSTORE_PUBLIC_URL_ROOT = 'http://localhost:18250'
 BLOCKSTORE_API_URL = 'http://localhost:18250/api/v1/'
-# Which of django's caches to use for storing anonymous user state for XBlocks
-# in the blockstore-based XBlock runtime
+
+# .. setting_name: XBLOCK_RUNTIME_V2_EPHEMERAL_DATA_CACHE
+# .. setting_default: default
+# .. setting_description: Which of django's caches to use for storing anonymous user state for XBlocks
```

Comment:
Taking from your PR description, what do you think of the following?:
```
# .. setting_description: The django cache key of the cache to use for storing anonymous user state for XBlocks.
```

Reply:
Sure that's more descriptive, I'll change it to that

Reply:
Ok. Ping me when this is updated and I can merge.  Thanks!

Reply:
@robrap this is updated now :+1: